spidermon/contrib/scrapy/pipelines.py

```diff
@@ -103,15 +103,16 @@ def process_item(self, item, _):
             # No validators match this specific item type
             return item
 
-        data = self._convert_item_to_dict(item)
+        item_adapter = ItemAdapter(item)
+        item_dict = item_adapter.asdict()
         self.stats.add_item()
-        self.stats.add_fields(len(list(data.keys())))
+        self.stats.add_fields(len(list(item_dict.keys())))
```

Comment:
**nitpick** we don't need to convert to `list`. We can use: `self.stats.add_fields(len(item_dict.keys()))`